src/storage/store/node_column.cpp

```diff
@@ -352,17 +389,30 @@ void NodeColumn::scan(transaction::Transaction* transaction, node_group_idx_t no
     scanUnfiltered(transaction, pageCursor, numValuesToScan, resultVector, offsetInVector);
 }
 
+// Page size must be aligned to 8 byte chunks for the 64-bit NullMask algorithms to work
+// without the possibility of memory errors from reading/writing off the end of a page.
+static_assert(PageUtils::getNumElementsInAPage(1, false) % 8 == 0);
```

Comment:
Add inline comment `/*requireNullColumn*/` after `false`. Same for L404.
We do this to simplify code reading without jumping to the function signature. (especially when reviewing code on github website)